src/modules/user/reducer.js

```diff
@@ -0,0 +1,19 @@
+import { handleActions } from 'redux-actions';
+import {
+  fetchUserDataSuccess,
+  fetchUserDataFailure,
+  userAuth,
+} from './actions';
+
+
+const user = handleActions({
+  [fetchUserDataSuccess]: (_state, action) => action.payload,
+  [fetchUserDataFailure]: (_state, action) => action.payload,
+  [userAuth]: (_state, action) => action.payload,
+}, {});
+
+
+export default user;
+
+
+export const getUser = state => state.user;
```

Comment:
these functions are called selectors and should go to a separate file called `selectors.js`